<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui::Pref::TDemuxer</class>
 <widget class="QWidget" name="Gui::Pref::TDemuxer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>400</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="demuxer_tabwidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="demuxer_tab">
      <attribute name="title">
       <string>Demuxer</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>16</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="icons_layout">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>16</number>
         </property>
         <item>
          <widget class="QLabel" name="icon1_label">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="icon2_label">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="icon3_label">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="icon4_label">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="icon5_label">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="lavf_demuxer_check">
         <property name="text">
          <string>Use the la&amp;vf demuxer by default</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="idx_check">
         <property name="text">
          <string>Rebuild &amp;index if needed</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
